packages/components/src/tooltip/index.js

```diff
@@ -253,7 +258,7 @@ function Tooltip( props ) {
 		: getRegularElement;
 
 	const popoverData = {
-		anchorRef: childRef,
+		anchor: popoverAnchor,
```

Comment:
What's the difference between components where we can simply pass `childRef.current` here, versus components that need a `useState` layer like this?

Reply:
Good question. I think that in theory all components using `Popover` should make sure that, if they internally use a ref to grab an anchor to pass to Popover, they re-render when the ref value updates.

We should probably assess that this is true in #43691, once we have all sub-PRs merged. I've added a todo item in the PR's description

Reply:
Ok. If this pattern is going to come up a lot, could we perhaps simplify it a little by removing the `useCallback` layer? So `const childRef = setPopoverAnchor;` (line 134) and then

```suggestion
		anchor: popoverAnchor ?? undefined,
```

Reply:
Good shout! Done in https://github.com/WordPress/gutenberg/pull/43799/commits/16d345ec1aaf03d0dccfc94b57c951345e87b74e